File: C++/First Steps in Coding - Basics of C++/Conditional Statements - Advanced/Fruit or Vegetable.cpp
#include <iostream>
#include <cstdlib>
#include <string>
using namespace std;
int main() {
	string prod;
	getline(cin, prod);
	if (prod == "banana" or prod == "apple" or prod == "kiwi" or prod == "cherry" or prod == "lemon" or prod == "grapes") {
		cout << "fruit";
	}
	else if (prod == "tomato" or prod == "cucumber" or prod == "pepper" or prod == "carrot") {
		cout << "vegetable";
	}
	else {
		cout << "unknown";
	}
}

File: Java/Advanced Conditional Statements/Lab/ACS Lab/src/FruitOrVegetable.java
import java.util.Scanner;

public class FruitOrVegetable {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String product = input.nextLine();
        String category;

        switch(product){
            case "banana":
            case "apple":
            case "kiwi":
            case "cherry":
            case "lemon":
            case "grapes":
                category = "fruit";
                break;
            case "tomato":
            case "cucumber":
            case "pepper":
            case "carrot":
                category = "vegetable";
                break;
            default:
                category = "unknown";
        }

        System.out.println(category);
    }
}
